### DISCUSSION TITLE
[Feature Request]: Is there a way to get the current power usage

### DISCUSSION THREAD

**Discussion by joaopedrocg27** [wlcrs/huawei_solar#826] [General]
### Describe your feature request

Maybe this is already possible but I was wondering if I could get the "instant" power usage from my house. I can get the inverter active power but this is only valid if the inverter is doing work. I can also get the meter but that does not include the inverter, etc.

I think I can get away with some calculations and conditions but probably there is a better way

Thank you!

### Proper usage

- [X] I confirm that this is not a bug report or support request
- [X] I confirm that this feature request is within the stated scope of the integration

**Comment by heinemannj** [comment]
Have a look on:

[Flows, Definitions and Calculations](https://github.com/heinemannj/huawei_solar_hems/wiki#flows-definitions-and-calculations)

||||
|---|---|---|
|House_Consumption_Power | W | = inv_in_power - bat_active_power - pm_active_power<br>= inv_in_power - (inv_in_power - inv_active_power) - pm_active_power<br>= inv_active_power - pm_active_power<br>= [32080] - [37113]|
